plugin.video.mlbtv/resources/lib/mlb.py

```diff
@@ -236,6 +238,7 @@ def stream_select(game_pk):
     else:
         xbmcplugin.setResolvedUrl(addon_handle, False, xbmcgui.ListItem())
         xbmc.executebuiltin('Dialog.Close(all,true)')
```

Comment:
why do you need to call this?

Reply:
It was my attempt to silence the "No playable stream found" on a setResolvedURL if the user hit cancel on the select dialog. I don't think it does what I wanted, so I can remove it.

Reply:
Yes please remove for merging